doc/hooks/installShellFiles.section.md

```diff
@@ -25,3 +25,17 @@ The `installShellCompletion` function takes one or more paths to shell completio
   '';
```

Comment:
L20 to 24 should probably be removed in favor of the next paragraph?

Reply:
Yeah, makes sense. Removed the duplicated code and moved the line about fifo/named fd closer to its actual example.